<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainFrame</class>
 <widget class="QMainWindow" name="CMainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZBridge</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionRecent_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Lay_Out_Cards"/>
    <addaction name="separator"/>
    <addaction name="actionCu_t"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_Expose_All_Cards"/>
    <addaction name="action_Score"/>
    <addaction name="actionDouble_Dummy_Results"/>
    <addaction name="separator"/>
    <addaction name="action_Toolbar"/>
    <addaction name="actionStatus_Bar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Session"/>
    <addaction name="action_Deal_New_Hand"/>
    <addaction name="actionActivate_Deal_Profile"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="action_Bid_Rebid"/>
    <addaction name="action_Restart_Hand"/>
    <addaction name="separator"/>
    <addaction name="actionClaim_All"/>
    <addaction name="actionClaim_Contract"/>
    <addaction name="actionConcede"/>
    <addaction name="separator"/>
    <addaction name="actionHint"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSeat_Configuration"/>
    <addaction name="action_Bidding_Options"/>
    <addaction name="action_Deal_Options"/>
    <addaction name="action_Di_splay_Options"/>
    <addaction name="action_Game_Options"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSpecial">
    <property name="title">
     <string>Special</string>
    </property>
    <addaction name="actionSelect_Bid_Database"/>
    <addaction name="actionEdit_Bid_Database"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_Contents"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_About_ZBridge"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGame"/>
   <addaction name="menuOptions"/>
   <addaction name="menuSpecial"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Session"/>
   <addaction name="action_Deal_New_Hand"/>
   <addaction name="actionActivate_Deal_Profile"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="action_Restart_Hand"/>
   <addaction name="action_Bid_Rebid"/>
   <addaction name="separator"/>
   <addaction name="actionHint"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionClaim_All"/>
   <addaction name="actionClaim_Contract"/>
   <addaction name="actionConcede"/>
   <addaction name="separator"/>
   <addaction name="action_Expose_All_Cards"/>
   <addaction name="separator"/>
   <addaction name="action_Score"/>
   <addaction name="actionDouble_Dummy_Results"/>
   <addaction name="separator"/>
  </widget>
  <widget class="CStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/FILEOPEN.BMP</normaloff>
     <normalon>:/newPrefix/resources/Open.ico</normalon>:/newPrefix/resources/FILEOPEN.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/FILESAVE.BMP</normaloff>
     <normalon>:/newPrefix/resources/Save.ico</normalon>:/newPrefix/resources/FILESAVE.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/Fileprint.bmp</normaloff>
     <normalon>:/newPrefix/resources/Print.ico</normalon>:/newPrefix/resources/Fileprint.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRecent_File">
   <property name="text">
    <string>Recent File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionCu_t">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/EDITCUT.BMP</normaloff>:/newPrefix/resources/EDITCUT.BMP</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/EDITCOPY.BMP</normaloff>:/newPrefix/resources/EDITCOPY.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/Editpaste.bmp</normaloff>:/newPrefix/resources/Editpaste.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Score">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/ViewScore.bmp</normaloff>:/newPrefix/resources/ViewScore.bmp</iconset>
   </property>
   <property name="text">
    <string> &amp;Score</string>
   </property>
   <property name="toolTip">
    <string>Score</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="action_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Toolbar</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="action_Deal_New_Hand">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/DealNewHand.bmp</normaloff>:/newPrefix/resources/DealNewHand.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Deal New Hand</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Bid_Rebid">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/BidHand.bmp</normaloff>:/newPrefix/resources/BidHand.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Rebid</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_Restart_Hand">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/RestartPlay.bmp</normaloff>:/newPrefix/resources/RestartPlay.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Replay</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionClaim_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/ClaimTricks.bmp</normaloff>:/newPrefix/resources/ClaimTricks.bmp</iconset>
   </property>
   <property name="text">
    <string>Claim All</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionClaim_Contract">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/ClaimContract.bmp</normaloff>:/newPrefix/resources/ClaimContract.bmp</iconset>
   </property>
   <property name="text">
    <string>Claim Contract</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="actionConcede">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/ConcedeTricks.bmp</normaloff>:/newPrefix/resources/ConcedeTricks.bmp</iconset>
   </property>
   <property name="text">
    <string>Concede</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionHint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/HelpAppAbout.bmp</normaloff>:/newPrefix/resources/HelpAppAbout.bmp</iconset>
   </property>
   <property name="text">
    <string>Hint</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="action_Lay_Out_Cards">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/LayOutCards.bmp</normaloff>:/newPrefix/resources/LayOutCards.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Lay Out Cards</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_Bidding_Options">
   <property name="text">
    <string>&amp;Bidding/Play...</string>
   </property>
  </action>
  <action name="action_Deal_Options">
   <property name="text">
    <string>&amp;Deal...</string>
   </property>
  </action>
  <action name="action_Di_splay_Options">
   <property name="text">
    <string>&amp;Di&amp;splay Options...</string>
   </property>
  </action>
  <action name="action_Game_Options">
   <property name="text">
    <string>&amp;Game...</string>
   </property>
  </action>
  <action name="action_Expose_All_Cards">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/ExposeAllCards.bmp</normaloff>:/newPrefix/resources/ExposeAllCards.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Expose All Cards</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Contents">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/HelpContents.bmp</normaloff>:/newPrefix/resources/HelpContents.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents</string>
   </property>
  </action>
  <action name="action_About_ZBridge">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/HelpAppAbout.bmp</normaloff>:/newPrefix/resources/HelpAppAbout.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;About ZBridge...</string>
   </property>
  </action>
  <action name="actionNew_Session">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/ManualDefend.bmp</normaloff>:/newPrefix/resources/ManualDefend.bmp</iconset>
   </property>
   <property name="text">
    <string>New Session</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/EDITUNDO.BMP</normaloff>:/newPrefix/resources/EDITUNDO.BMP</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo bid or trick</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionActivate_Deal_Profile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/numbered.bmp</normaloff>:/newPrefix/resources/numbered.bmp</iconset>
   </property>
   <property name="text">
    <string>Activate Deal Profile</string>
   </property>
  </action>
  <action name="actionSeat_Configuration">
   <property name="text">
    <string>Seat Configuration</string>
   </property>
  </action>
  <action name="actionDouble_Dummy_Results">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/newPrefix/resources/ViewPlayHistory.bmp</normaloff>:/newPrefix/resources/ViewPlayHistory.bmp</iconset>
   </property>
   <property name="text">
    <string>Double Dummy Results</string>
   </property>
  </action>
  <action name="actionSelect_Bid_Database">
   <property name="text">
    <string>Select Bid Database</string>
   </property>
  </action>
  <action name="actionEdit_Bid_Database">
   <property name="text">
    <string>Edit Bid Database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CStatusBar</class>
   <extends>QStatusBar</extends>
   <header>cstatusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Toolbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatus_Bar</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
